openedx/core/djangoapps/user_api/accounts/tests/test_helpers.py

```diff
@@ -0,0 +1,54 @@
+from ddt import ddt, data
+import hashlib
+from mock import patch
+
+from django.test import TestCase
+
+from openedx.core.djangoapps.user_api.accounts.helpers import get_profile_image_url_for_user
+from student.tests.factories import UserFactory
+
+from .utils import ProfileImageUrlTestCaseMixin
+
+
+@ddt
+@patch.dict(
+    'openedx.core.djangoapps.user_api.accounts.helpers.PROFILE_IMAGE_SIZES', {'full': 50}, clear=True
+)
+class ProfileImageUrlTestCase(ProfileImageUrlTestCaseMixin, TestCase):
+    """
+    Tests for `get_profile_image_url_for_user`.
+    """
+    def setUp(self):
+        self.user = UserFactory()
+
+    def test_has_no_profile_image(self):
+        """
+        Verify we get the URL to the default image if the user does not
+        have a profile image.
+        """
+        self.user.has_profile_image = False
```

Comment:
If we create a new user, we expect has_profile_mage to be False. Let's test that.

Then test setting it to True, and then back to False. I feel like you should be saving though -- user.profile.save().